trying to merge again

molly-100 opened the pull request:
i am sending a pull request to hadassah so hadassah  can review the chnages and fix the main file we are using

molly-100 committed:
Merge branch 'master' of https://github.com/HadassahJoseph/MusicVisuals

molly-100 committed:
updated comments

molly-100 committed:
trying to fix files

molly-100 committed:
Merge branch 'master' of https://github.com/HadassahJoseph/MusicVisuals

molly-100 committed:
check if file is working after merge

molly-100 committed:
make the notes move up and down vertically

molly-100 committed:
fixed movement of notes

molly-100 committed:
added array list for colours

molly-100 committed:
changing colour of notes

molly-100 committed:
added to drawNotes()

molly-100 committed:
updated update method

molly-100 committed:
updated draw

molly-100 committed:
working version

molly-100 committed:
update

molly-100 committed:
update comments

molly-100 committed:
add new note

molly-100 committed:
add to draw

molly-100 committed:
fix note

molly-100 committed:
added to note

molly-100 committed:
draw another note

molly-100 committed:
notes drawing fixed

HadassahJoseph commented on java/src/ie/tudublin/music_note.java:
This is moll new code for her file and it has music nites and faling starts which i will add to her folder

HadassahJoseph committed:
Deleting the experiance file 

I do not need this file as it is already on the main file and i am only trying to accept her music note file

HadassahJoseph committed:
deleting main 

I do not need this file as it is already on the main file and i am only trying to accept her music note file

HadassahJoseph reviewed (approved):
trying to send only molly's music _notes file
